varinfo/umm_var.py

```diff
@@ -338,3 +341,53 @@ def get_json_serializable_value(input_value: Any) -> Any:
         output_value = input_value
 
     return output_value
+
+
+def publish_umm_var(collection_id: str,
+                    umm_var_dict: Dict,
+                    token: str,
+                    cmr_env: CMR_UAT) -> str:
+    """" Publish a single UMM-Var entry to CMR given:
+        * collection_id: a collection's concept_id
+        * umm_var_dict: a dictionary of a single UMM-Var entry for a collection
+        * token: Earthdata Login (EDL) bearer_token
+        * cmr_env/mode: CMR environments (OPS, UAT, and SIT)
+    For a successful publication response all of these fields must be entered
+
+    """
+    # Required UMM-Var headers for ingestiong variable entries
+    headers_umm_var = {
+        'Content-type': 'application/vnd.nasa.cmr.umm+json;version='
+        + f'{umm_var_dict["MetadataSpecification"]["Version"]}',
+        'Authorization': f'Bearer {token}',
+        'Accept': 'application/json'}
+
+    url_endpoint = (cmr_env.replace('search', 'ingest') + 'collections/'
+                    f'{collection_id}/variables/{umm_var_dict["Name"]}')
+
+    response = requests.put(url_endpoint,
+                            json=umm_var_dict,
+                            headers=headers_umm_var,
+                            timeout=10)
+    # Check the status_code of the response
+    # Return the variable concept-id if response is 200
+    if response.status_code == 200:
```

Comment:
Nitpick: It might be a tiny bit more flexible to make the check `if response.ok`, which will return true for any 200 type status code (200, 201, etc).